src/_utils/icon/index.tsx

```diff
@@ -8,6 +9,11 @@ const StyledBaseIcon = styled(BaseIcon)`
     position: relative;
   }
 
+  &.kirk-icon-shadowed {
```

Comment:
I wanted to add the style only for `MeetingPointIcon`, but we break our stories doing so (styled HOC prevents access to defaultProps)